internal/controller/replication.storage/status.go

```diff
@@ -120,18 +133,21 @@ func setNotDegradedCondition(conditions *[]metav1.Condition, observedGeneration
 // sets conditions when volume was demoted successfully.
 func setDemotedCondition(conditions *[]metav1.Condition, observedGeneration int64) {
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is demoted",
 		Type:               v1alpha1.ConditionCompleted,
 		Reason:             v1alpha1.Demoted,
 		ObservedGeneration: observedGeneration,
 		Status:             metav1.ConditionTrue,
 	})
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is Degraded",
```

Comment:
Mostly the state is written lowercase. Make sure you use the same formatting everywhere (whichever you prefer).